src/components/PlayingScreen.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import Board from './Board';
+import VideoArea from './VideoArea';
+import { StoreState } from '../types/index';
+import { connect } from 'react-redux';
+
+const mapStateToProps = (state: StoreState) => {
+  const selectedGameId = state.activeGame;
```

Comment:
There is no state.activeGame :)
You meant:
  currentMatchIndex: number; // an index in matchesList